sql/init_data/init_core.sql

```diff
@@ -42,7 +42,7 @@ INSERT INTO INT2_TBL(f1) VALUES ('  1234 ');
 --Testcase 3:
 INSERT INTO INT2_TBL(f1) VALUES ('    -1234');
 --Testcase 4:
-INSERT INTO INT2_TBL(f1) VALUES ('34.5');
+INSERT INTO INT2_TBL(f1) VALUES ('345');
```

Comment:
Could you explain this change.

Reply:
This is invalid value for `INT2_TBL` tests. This value doesn't belongs to `int2` or `smaillint` domain. This value doesn't used as tested value in any place. No data of other domains tested around of  `INT2_TBL` like around  `INT4_TBL` and some similar tests. I think this data value was error.

Reply:
Thank for explaining. However, this SQL create based on PostgreSQL regression test (some invalid data). Please revert it.

Reply:
For SQLite this is not invalid data... This is invalid data only for PostgreSQL. SQLite mark this data as `float` affinity. Hence there is no TC object.

Reply:
OK thank for explaining.

Reply:
@nxhai98 , should I revert this change anyway?

Reply:
I think it had better to revert this change. We will change it later if we use `INT2_TBL` in future

Reply:
I have tested and ensure this revert cause errors in  `lateral join` test (`Testcase 309`) because of type mismatch. Maybe this is a new error like with `mod()`?
```
--Testcase 309:
 select * from int4_tbl i left join
   lateral (select * from int2_tbl j where i.f1 = j.f1) k on true;
-     f1      | f1 
--------------+----
- -2147483647 |   
-     -123456 |   
-           0 |  0
-      123456 |   
-  2147483647 |   
-(5 rows)
-
+ERROR:  SQLite data affinity "real" disallowed for PostgreSQL data type "smallint"
+HINT:  In column "f1" expected SQLite affinity "integer", incorrect value = '34.5'
```

Reply:
@mkgrgis, For this issue, I discussed with @t-kataym -san and decide to remove invalid value for int2: `34.5`   
Please help me to remove this SQL and update expected result.

Reply:
Fixed in https://github.com/mkgrgis/sqlite_fdw/commit/b10f047dc8f7ca6cf98d2f00bddd5afd1298d320, @nxhai98 .

Reply:
Thanks for your fix. I confirmed.

Reply:
Resolved.